/**:
  ros__parameters:
    dynamics:
      world_box:
        restitution: 0.3
        lateralFriction: 0.5
        spinningFriction: 0.3
        contactDamping: 10000.0
        contactStiffness: 1000000.0
      pedestal:
        restitution: 0.3
        lateralFriction: 0.5
        spinningFriction: 0.3
        contactDamping: 10000.0
        contactStiffness: 1000000.0
      
      